src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToReferenceAProject.cs

```diff
@@ -152,5 +153,86 @@ TestProject GetTestProject(string name, string target, bool isSdkProject)
 
             return ret;
         }
+
+        [CoreMSBuildOnlyTheory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void It_disables_copying_conflicting_transitive_content(bool copyConflictingTransitiveContent)
```

Comment:
Should we also cover the case where the property is implicitly vs explicitly set to false?